services/auth/src/api/routes/profile.py

```diff
@@ -0,0 +1,73 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from api.deps import AccessRequire, Session
+from models import User, Profile
+from schemas import MeResponse, ProfileSchema, ProfileUpdate
+
+router = APIRouter(prefix="/me", tags=["Profile"])
+
+
+@router.get("", response_model=MeResponse)
+async def get_me(
+    require: AccessRequire,
+    session: Session,
+) -> MeResponse:
+    """Get current user with profile."""
+    user_id = require.payload["sub"]
+    
+    # Load user with profile
+    result = await session.execute(
+        select(User).where(User.id == user_id)
+    )
+    user = result.scalar_one_or_none()
+    
+    if not user:
+        from fastapi import HTTPException, status
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="User not found",
+        )
+    
+    # Create profile if doesn't exist
+    if not user.profile:
+        profile = Profile(user_id=user.id)
+        session.add(profile)
+        await session.commit()
+        await session.refresh(user)
```

Comment:
Вот тут как раз, да, нужно зафлушить вместо коммита и рефреша, так как это создание объекта в базе, и в `MeResponse` уже должен попасть `user.profile` из базы.
```suggestion
        await session.flush()
```